tasks/js-oop.md

```diff
@@ -28,22 +28,10 @@ When complete do the following:
 1. You did a lot already! If you honestly finished all the previous steps then go ahead
```

Comment:
IMO, this part can be moved elsewhere because it is repeated from task to task? 🤔